templates/obsidian/reference.md

```
@@ -0,0 +1,41 @@
+if (scope === "project") {
+  const pf = app.vault.getAbstractFileByPath("projects");
+  const existing = pf ? pf.children.filter(f => f.children).map(f => f.name).sort() : [];
+  const options = [...existing, "＋ New project…"];
+  const sel = await tp.system.suggester(options, options, false, "Select project");
+  project = sel === "＋ New project…" ? await tp.system.prompt("Project name (slug)") : sel;
+}
+const title = await tp.system.prompt("Reference title");
+const summary = await tp.system.prompt("Describe this reference");
+const sourceUrl = await tp.system.prompt("Source URL (leave empty if none)", "");
+const ts = tp.file.title.slice(0, 16);
+const dir = project ? `projects/${project}/references` : "general/references";
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)